dali/util/uri.h

```diff
@@ -47,7 +47,25 @@ class URI {
   }
 
  public:
-  static DLL_PUBLIC URI Parse(std::string uri);
+  /**
+   * @brief Parse options
+   * 
+   */
```

Comment:
Nitpick:
```suggestion
  /** Parse options */
```
^^^ This is also a "brief" description, but takes much fewer lines.